prog/weaver/main.go

```diff
@@ -441,6 +442,7 @@ func main() {
 		HandleHTTP(muxRouter, version, router, allocator, defaultSubnet, ns, dnsserver, &waitReady)
 		HandleHTTPPeer(muxRouter, allocator, discoveryEndpoint, token, name.String())
 		muxRouter.Methods("GET").Path("/metrics").Handler(metricsHandler(router, allocator, ns, dnsserver))
+		muxRouter.Methods("GET").Path("/proxyaddrs").HandlerFunc(proxy.StatusHTTP)
```

Comment:
Interestingly `weave status` doesn't indicate the problem, since `weaver` can just crash and restart.